config/core/test-configs.yaml

```diff
@@ -2622,6 +2622,12 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - kselftest-cpufreq
+      - kselftest-filesystems
```

Comment:
OK https://staging.kernelci.org/test/plan/id/62656611fc3358838d178696/

Failed with Clang: https://staging.kernelci.org/test/plan/id/62656bb2e371c78736178697/

Reply:
As discussed on the Juno pull request I don't think the clang kselftest builds are actually cross compiling anything, it looks like clang kselftest builds should be disregarded in terms of working on machines.